tools/circle_plus_gen/README.md

```diff
@@ -0,0 +1,40 @@
+# Circle+ generator
+
+If a circle file has training hyperparameters, we usually call it a 'circle+' file.<br/>
+This tool generates a circle+ file by injecting training hyperparameters into a circle file.<br/>
+It also helps handle circle+ file, such as checking whether the circle file contains training hyperparameters. <br/> 
+
+## Requirements
+
+1. (optional) Set python virtaul environment.
+
+    This tool tested on python3.8.
+
+    ```
+    python3 -m venv venv
+    source /venv/bin/activate
+    ```
+
+2. Install required pakcages. 
+
+    Currently, only `flatbuffers==24.3.25` is needed.
+    ```bash
+    pip install -r requirements.txt
```

Comment:
I guess this tool requires python3.8, then it would be nice to specify python version when installing pip packages.
```suggestion
    python3 -m pip install -r requirements.txt
```